Formula/fasttext.rb

```diff
@@ -0,0 +1,18 @@
+class Fasttext < Formula
+  desc "Library for fast text representation and classification"
+  homepage "https://fasttext.cc"
+  url "https://github.com/facebookresearch/fastText/archive/v0.2.0.tar.gz"
+  sha256 "71d24ffec9fcc4364554ecac2b3308d834178c903d16d090aa6be9ea6b8e480c"
+  head "https://github.com/facebookresearch/fastText.git"
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"trainingset").write("__label__brew brew")
+    system "#{bin}/fasttext", "supervised", "-input", "trainingset", "-output", "model"
```

Comment:
Can we check that the output file has been created?

Reply:
Verified that the model is created in c091b4d